index.js

```diff
@@ -455,51 +455,51 @@ async function sendTwilioMessage(fromPhone, toPhone, msg) {
     }
 }
 
-
 // TODO: replace these many almost identical functions with something more elegant
 async function sendInitialChatbotMessage(session) {
     console.log("Intial message sent");
     var location = session.locationid;
     var alertReason = session.alert_reason;
     let locationData = await db.getLocationData(location);
-    await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `This is a ${alertReason} alert. Please check on the bathroom. Please respond with 'ok' once you have checked on it.`);
     await db.startChatbotSessionState(session);
+    await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `This is a ${alertReason} alert. Please check on the bathroom. Please respond with 'ok' once you have checked on it.`);
     setTimeout(reminderMessage, locationData.unresponded_timer, session.sessionid);
     setTimeout(fallbackMessage, locationData.unresponded_session_timer, session.sessionid)
 }
 
 async function reminderMessage(sessionid) {
-    console.log("Reminder message being sent");
-    let session = await db.getSessionWithSessionId(sessionid); // Gets the updated state for the chatbot
-    if(session.chatbot_state == STATE.STARTED) {
-        //Get location data
-        var location = session.locationid;
-        let locationData = await db.getLocationData(location)
-        //send the message
-        await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `This is a reminder to check on the bathroom`)
-        session.chatbot_state = STATE.WAITING_FOR_RESPONSE;
-        let chatbot = new Chatbot(session.sessionid, session.locationid, session.chatbot_state, session.phonenumber, session.incidenttype, session.notes);
-        await db.saveChatbotSession(chatbot);
+    if(!helpers.isTestEnvironment()){
```

Comment:
To avoid having this function be called in the integration tests, could we instead mock it in the test and not having this `isTestEnvironment()` check?

Reply:
I am actually unsure about how to go about this, maybe we could pair again on Thursday and you could help me mock it?

Reply:
Sure, that sounds good to me. This afternoon?